Kentico.Kontent.AspNetCore/Webhooks/Models/DeliveryWebhookModel.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace Kentico.Kontent.AspNetCore.Webhooks.Models
+{
+    /// <summary>
+    /// Root object of a Kontent delivery or preview-delivery API triggered webhook.
```

Comment:
There might be a link to API reference (optional).

Reply:
Added webhooks reference documentation link.